sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -455,7 +455,7 @@ public void releaseReferences() {
     }
   }
 
-  private static DType toRapidsOrNull(DataType type) {
+  private static DType toNonNestedRapidsOrNull(DataType type) {
```

Comment:
This method says it will return a non-nested RAPIDS type or null, yet it will return `LIST` for `BinaryType` which seems inconsistent.

Reply:
Is it clearer now? Or I can just revert it back to what it was.